README.md

```diff
@@ -1,6 +1,141 @@
+Alternatively you can [install](#-installation) the `evy` toolchain
+locally.
+
+Here is a "hello world" program in Evy
+
+    print "Hello World!"
+
+<details>
+  <summary>Screencast: How to code a simple animation in Evy.</summary>
+
+[![Coding evy](docs/img/purple-dot.gif)](https://evy.dev)
+
+[Animation source code]
+
+</details>
+
+[evy.dev]: https://evy.dev
+[Animation source code]: https://evy.dev/#content=H4sIAAAAAAAAEzWLwQqAIBBE7/sVg/fSiC6BHyO2B0FXWazvz4qGGXjMMKVejM09thaRpbNSrLkqTDu1ZTak2D1WRzFpzAwlqoIgqYTOhKHRBn1J4UcmuHn5lv/CctAN/HT8mWwAAAA=
+
+## 📖 Documentation
+
+Here are some resources for learning more about Evy:
+
+- [Learn and Practise](https://github.com/foxygoat/evy/wiki): A guided tour that teaches you programming using Evy.
+- [Syntax by Example](docs/syntax_by_example.md): A collection of examples that illustrate the Evy syntax.
+- [Built-in Documentation](docs/builtins.md): Details on built-in functions and events in Evy.
+- [Language Specification](docs/spec.md): A formal definition of the Evy syntax and language.
+
+## 📦 Installation
+
+You can install the Evy toolchain locally to run it from your command
+line. To learn how to use the Evy toolchain, read the
+[usage documentation](docs/usage.md) or run `evy --help`.
+
+### Linux and Windows
+
+Download the [latest release] for your platform, unzip it and add `evy`
+to your path.
+
+### macOS
+
+Use [Homebrew] to install `evy`.
+
+    brew install foxygoat/tap/evy
+
+[latest release]: https://github.com/foxygoat/evy/releases/latest
+[Homebrew]: https://brew.sh/
+
+## 💻 Development
+
+The Evy compiler is written in [Go] and built using the Go and
+[TinyGo] compilers. TinyGo targets [WebAssembly], which allows Evy
+source code to be compiled and run in a web browser. The browser
+runtime is written in plain JavaScript without the use of frameworks.
+
+To build the Evy source code, [clone] this repository and
+[activate Hermit] in your terminal. Then, build the sources with
+
+    make all
+
+You can list make targets, execute a full CI run locally or serve the
+frontend locally with
+
+    make help
+    make ci
+    make frontend-serve
+
+<details>
+  <summary>Activated Hermit auto-installs pinned tools.</summary>
+
+### Hermit
+
+The tools used in this repository, such as Make, Go and Node, are
+automatically downloaded by [Hermit] when needed. Hermit ensures that
+developers on Mac, Linux, and GitHub Actions CI use the same version of
+the same tools. Cloning this repo is the only installation step
+necessary.
+
+There are two ways to use the tools in the Evy repository. You can
+either prefix them with `bin/`, for example `bin/make all`. Or, you can
+activate Hermit in your shell with
+
+    . ./bin/activate-hermit
+
+This will add the tools to your path, so you can use them without having
+to prefix them with `bin/`.
+
+You can auto-activate Hermit when changing into the `evy` source
+directory by installing Hermit shell hooks with
```

Comment:
I would link "Hermit shell hooks" to https://cashapp.github.io/hermit/usage/shell/#shell-hooks

Reply:
great idea, thank.